pkg/config/install_requirements.go

```diff
@@ -473,6 +489,8 @@ type RequirementsConfig struct {
 	AutoUpdate AutoUpdateConfig `json:"autoUpdate,omitempty"`
 	// BootConfigURL contains the url to which the dev environment is associated with
 	BootConfigURL string `json:"bootConfigURL,omitempty"`
+	// BuildPackConfig contains custom build pack settings
+	BuildPacks BuildPackConfig `json:"buildPacks,omitempty"`
```

Comment:
Actually, this should be a pointer too.

Reply:
I think this being the pointer is key

Reply:
Yeah, but it's best practices to make any `omitempty` field that's not a string/bool/int/etc be a pointer anyway. =)

Reply:
> Yeah, but it's best practices to make any omitempty field that's not a string/bool/int/etc be a pointer anyway.

agreed. All should be pointers really. The GitHub lib does a better job there.